src/main/java/uk/gov/digital/ho/hocs/casework/api/SomuItemResource.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.digital.ho.hocs.casework.api;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.digital.ho.hocs.casework.api.dto.*;
+import uk.gov.digital.ho.hocs.casework.domain.model.*;
+
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Slf4j
+@RestController
+public class SomuItemResource {
+
+    private final SomuItemService somuItemService;
+
+    @Autowired
+    public SomuItemResource(SomuItemService somuItemService) {
+        this.somuItemService = somuItemService;
+    }
+    
+    @GetMapping(value = "/case/{caseUuid}/item")
```

Comment:
Would be better to tell it that it produces JSON. Same for line 31.

Reply:
Agreed - I will add it.